lib/Controller/ReportController.php

```diff
@@ -70,4 +72,29 @@ public function __construct(
 	public function getReport() {
 		return new ReportResponse('', '', $this->reportDataCollector->getReportJson());
 	}
+
+	/**
+	 * API-Handler which is used to get web config-report from cli. Auth-Token
+	 * is generated by the cli-app for onetime use.
+	 *
+	 * @NoCSRFRequired
+	 * @NoAdminRequired
+	 * @PublicPage
+	 *
+	 * @return ReportResponse|DataResponse with the report
+	 */
+	public function fromCli() {
+		// token generated by cli app
+		$actualToken = $this->config->getAppValue('configreport', 'token');
+		$passedToken = $this->request->getParam('token');
+
+		if (($actualToken !== $passedToken) || (empty($passedToken) || empty($actualToken))) {
```

Comment:
from a security perspective, we should log an error here. It's interesting to know if there has been a failed attempt.